velox/functions/prestosql/tests/DecimalArithmeticTest.cpp

```diff
@@ -379,6 +379,55 @@ TEST_F(DecimalArithmeticTest, round) {
           DECIMAL(38, 1))});
 }
 
+TEST_F(DecimalArithmeticTest, roundN) {
+  // Round upto 'scale' decimal places.
+  testDecimalExpr<TypeKind::BIGINT>(
+      {makeShortDecimalFlatVector({123, 552, -999, 0}, DECIMAL(4, 3))},
+      "round(c0, CAST(3 as integer))",
+      {makeShortDecimalFlatVector({123, 552, -999, 0}, DECIMAL(3, 3))});
+  // Round upto scale-1 decimal places.
+  testDecimalExpr<TypeKind::BIGINT>(
+      {makeShortDecimalFlatVector({120, 550, -1000, 0}, DECIMAL(4, 3))},
+      "round(c0, CAST(2 as integer))",
+      {makeShortDecimalFlatVector({123, 552, -999, 0}, DECIMAL(3, 3))});
+  // Round upto 0 decimal places.
+  testDecimalExpr<TypeKind::BIGINT>(
+      {makeShortDecimalFlatVector({100, 600, -1000, 0}, DECIMAL(4, 2))},
+      "round(c0, CAST(0 as integer))",
+      {makeShortDecimalFlatVector({123, 552, -999, 0}, DECIMAL(3, 2))});
+  // Round upto -1 decimal places.
+  testDecimalExpr<TypeKind::BIGINT>(
+      {makeShortDecimalFlatVector({120, 550, -1000, 0}, DECIMAL(4, 1))},
+      "round(c0, CAST(0 as integer))",
```

Comment:
should be `-1`?

Reply:
```
presto> select round(decimal'12.3',0);
 _col0 
-------
 12.0  
(1 row)

presto> select round(decimal'55.2',0);
 _col0 
-------
 55.0  
(1 row)

presto> select round(decimal'-99.9',0);
 _col0  
--------
 -100.0 
(1 row)
```

Reply:
The comment says `// Round upto -1 decimal places.`. But the test uses `round(c0, CAST(0 as integer))`?

Reply:
@karteekmurthys can you address this comment?